sections/07_conclusions.md

```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
 scientists to ask questions that they didn't know how to ask, it will be clear
 that deep learning has transformed biology and medicine.
 
+### Continuous collaborative manuscript drafting
+
+We recognized that deep learning in precision medicine is a rapidly developing area.
+Hence, diverse expertise was required to provide a forward-looking perspective.
+Accordingly, we collaboratively wrote this review in the open, enabling anyone with expertise to contribute.
+We wrote the manuscript in markdown and tracked changes using git.
+Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
+
+To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
```

Comment:
I use URLs knowing that they will likely be removed by the journal. However, there is still value added in the interim. I will make sure some of these URLs shouldn't also be references. So the hyperlink policy is that they the text is sufficient without them. Do you think that's a good solution?